elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -21,6 +21,7 @@ $variables: (
 
   [tabindex] {
   	display: flex;
+    height: 100%;
```

Comment:
@starryeyez024 Maybe this is a trick of the preview tool but it looks like the indention in this file is a bit off.

Reply:
adjusted